book/src/keymap.md

```diff
@@ -363,9 +363,9 @@ Keys to use within picker. Remapping currently not supported.
 
 | Key                          | Description       |
 | -----                        | -------------     |
-| `Up`, `Ctrl-p`               | Previous entry    |
+| `Tab`, `Up`, `Ctrl-p`               | Previous entry    |
 | `PageUp`, `Ctrl-u`           | Page up           |
-| `Down`, `Ctrl-n`             | Next entry        |
+| `Shift-tab`, `Down`, `Ctrl-n`             | Next entry        |
 | `PageDown`, `Ctrl-d`         | Page down         |
```

Comment:
Can you fix the spacing in the table here so the bars are aligned?

Reply:
Yes, I’m going to sleep now but I’ll do it tomorrow